prime-router/src/main/kotlin/fhirengine/utils/FHIRBundleHelpers.kt

```diff
@@ -445,7 +448,7 @@ fun Bundle.enhanceBundleMetadata(hl7Message: Message) {
         else -> ""
     }
     this.identifier.value = identifierValue
```

Comment:
nit: Any reason not to set this directly with the preceding when statement (line 441/444)

Reply:
No that I could find, done!